CDP4JsonSerializer/Helper/SerializerHelper.cs

```diff
@@ -24,34 +24,58 @@
 
 namespace CDP4JsonSerializer
 {
+    using System;
     using System.Collections.Generic;
+    using System.Linq;
+    using System.Text.RegularExpressions;
 
     using CDP4Common.Types;
-    using CDP4Common.Helpers;
+
+    using Newtonsoft.Json;
     using Newtonsoft.Json.Linq;
     
-    using NLog;
-
     /// <summary>
     /// Utility method to convert a JSON token to a CDP4 type
     /// </summary>
     public static class SerializerHelper
     {
+        /// <summary>
+        /// Regex used for conversion of Json value to string
+        /// </summary>
+        private static readonly Regex JsonToValueArrayRegex = new Regex(@"^\[(.*)\]$", RegexOptions.Singleline);
+        
+        /// <summary>
+        /// Regex used for conversion of HStore value to string
+        /// </summary>
+        private static readonly Regex HstoreToValueArrayRegex = new Regex(@"^\{(.*)\}$", RegexOptions.Singleline);
+
+        /// <summary>
+        /// Convert a string to a <see cref="ValueArray{T}"/>
+        /// </summary>
+        /// <typeparam name="T">The generic type of the <see cref="ValueArray{T}"/></typeparam>
+        /// <param name="valueArrayString">The string to convert</param>
+        /// <returns>The <see cref="ValueArray{T}"/></returns>
+        public static ValueArray<T> FromHstoreToValueArray<T>(string valueArrayString) =>
+            ToValueArray<T>(valueArrayString, HstoreToValueArrayRegex);
+
         /// <summary>
         /// Convert a string to a <see cref="ValueArray{T}"/>
         /// </summary>
         /// <typeparam name="T">The generic type of the <see cref="ValueArray{T}"/></typeparam>
         /// <param name="valueArrayString">The string to convert</param>
         /// <returns>The <see cref="ValueArray{T}"/></returns>
-        public static ValueArray<T> ToValueArray<T>(string valueArrayString) => ValueArrayUtils.FromJsonToValueArray<T>(valueArrayString);
+        public static ValueArray<T> ToValueArray<T>(string valueArrayString) => ToValueArray<T>(valueArrayString, JsonToValueArrayRegex);
 
         /// <summary>
         /// Convert a <see cref="ValueArray{String}"/> to the JSON format
         /// </summary>
         /// <param name="valueArray">The <see cref="ValueArray{String}"/></param>
         /// <returns>The JSON string</returns>
-        public static string ToJsonString(this ValueArray<string> valueArray) =>
-            ValueArrayUtils.ToJsonString(valueArray);
+        public static string ToJsonString(ValueArray<string> valueArray)
```

Comment:
we could make this an extension method: `public static string ToJsonString(this ValueArray<string> valueArray)`